src/particles/box/documentation.stories.mdx

```diff
@@ -75,6 +78,26 @@ export const Template = (args) => (
   </Story>
 </Canvas>
 
+<Canvas>
+  <Story name='Clipped Content'>
+    <Box width='125px' height='100px'>
```

Comment:
once you replace the pill with a black overlay i would hope this example does not work. you would need to add the clipping to this box.